app/advanced-analytics/custom-dashboards-reference.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Custom Dashboards reference"
```

Comment:
Should we add anything about automation? Users can either use the API endpoint + the "Download definition as JSON" option (this is a feature in the UI that gives you the entire dashboard definition as a JSON file) or Terraform.